src/tester/upload.py

```diff
@@ -83,9 +81,6 @@ def main():
     )
     storage_client = storage.Client(PROJECT_ID, credentials=credentials)
     bucket = storage_client.bucket("rubin-prompt-proto-main")
-    batch_settings = pubsub_v1.types.BatchSettings(
-        max_messages=INSTRUMENTS[instrument].n_detectors,
-    )
     publisher = pubsub_v1.PublisherClient(credentials=credentials)
```

Comment:
@ktlim, is this what it should look like? Google's documentation is shockingly bad.
```suggestion
    publisher = pubsub_v1.PublisherClient(credentials=credentials, per_partition_batching_settings=batch_settings)
```

Reply:
See https://googleapis.dev/python/pubsub/latest/publisher/index.html#batching
and https://googleapis.dev/python/pubsub/latest/publisher/api/client.html#module-google.cloud.pubsub_v1.publisher.client

I think this is just `batch_settings=batch_settings`.

Reply:
Oops, wrong namespace: I was looking at https://cloud.google.com/python/docs/reference/pubsublite/latest/google.cloud.pubsublite.cloudpubsub.publisher_client.PublisherClient.

Reply:
https://cloud.google.com/pubsub/docs/choosing-pubsub-or-lite